_includes/openseeds-header.html

```diff
@@ -0,0 +1,78 @@
+<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
+    <div class="container">
+      <div class="navbar-brand">
+        <a class="navbar-item" href="{% link openseeds/index.md %}">
+            <img src="/images/logo.png" alt="OLS">
+        </a>
+        <a
+          role="button"
+          class="navbar-burger burger"
+          aria-label="menu"
+          aria-expanded="false"
+          data-target="top-menu"
+        >
+          <span aria-hidden="true"></span> <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+        </a>
+      </div>
+
+      <div id="top-menu" class="navbar-menu">
+        <div class="navbar-end">
+          <div class="navbar-item has-dropdown is-hoverable">
+            <div class="navbar-dropdown">
+              <a class="navbar-item" href="{% link openseeds/index.md %}#applications"> Applications </a>
+              <a class="navbar-item" href="{% link about.md %}"> The program itself </a>
```

Comment:
The `about.md` page should move to the `openseeds`

Reply:
Done. Thanks!